CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+## Contributing
+
+1. Fork it
+2. Create your feature branch from `gh-pages` (`git checkout -b my-new-feature gh-pages`)
+3. Commit your changes (`git commit -am 'Topic: Changes made'`)
```

Comment:
I would remove the `-a`, only because that adds all files -- some people may end up committing dotfiles here